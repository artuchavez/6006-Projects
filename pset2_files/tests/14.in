gradebook-large
10
Robbin Barris,Valery Desper,Cherlyn Eldred,Todd Vegas,Carolynn Olague,Rafael Horsley,Kali Stockdale,Ellsworth Escalera,Mohammed Abad,Marla Midgley,Nicolasa Macinnis,Lashawna Royer,Cammy Schwartzberg,Pearline Pinette,Fausto Vanduzer,Ardelle Kapoor,Tiana Sutera,Mozell Tilson,Kali Vencill,Tamatha Chuang
Mohammed Abad,Nicolasa Macinnis,Tiana Sutera,Lashawna Royer,Kali Stockdale,Tamatha Chuang,Lashawna Royer,Pearline Pinette,Ellsworth Escalera,Kali Vencill
3,1,6,12,1,10,5,1,12,3
0.89,0.78,4.77,4.79,4.33,1.91,4.18,0.72,0.35,0.32
