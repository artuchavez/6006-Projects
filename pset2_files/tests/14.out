Mozell Tilson,Robbin Barris,Cammy Schwartzberg,Kali Vencill,Pearline Pinette,Valery Desper,Lashawna Royer,Fausto Vanduzer,Todd Vegas,Carolynn Olague,Rafael Horsley,Ardelle Kapoor,Kali Stockdale,Tiana Sutera,Cherlyn Eldred,Ellsworth Escalera,Tamatha Chuang,Mohammed Abad,Marla Midgley,Nicolasa Macinnis
0,0,0,0.32,0.72,0,4.61058823529,0,0,0,0,0,4.33,4.77,0,0.35,1.91,0.89,0,0.78
0,0,0,0,0,0,0.32,0.35,0.72,0.78